Scripts/General/input_manager.gd

```diff
@@ -170,6 +170,20 @@ func set_game_paused(is_paused : bool):
 func quit_game():
 	get_tree().quit()
 
+
+func get_player_by_index(index : int) -> Player:
+	if index in range(players.size()):
+		return players[index]
+	return null
```

Comment:
this could be an assert, but i assymed that nautral player is -1 (and as a fallback --everything with invalid player is neutral -- maybe it is not good idea, but for now i assumed it is enough)